scripts/install-hugo.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
why not simply do `brew install hugo`?  If someone already has it, this script might overwrite the binary, making brew unhappy.

Reply:
Well, the issue is that I’d prefer to avoid version discrepancies between different local environments and between local environments and the Netlify version. Hugo is still pre 1.0 and occasionally makes breaking changes.

Reply:
Doesn't brew support pinning the version?

Anyway, direct install is not all bad, but I would make it conditional on hugo not installed at all. If it is installed, I would add a version check and fail with a descriptive message on how to fix it.

Reply:
Unfortunately, Homebrew simply [doesn't support](https://stackoverflow.com/questions/3987683/homebrew-install-specific-version-of-formula) anything resembling an easily automatable `brew install TOOL VERSION`.

As for a version checking script, it's a bit tricky because the version output looks like this:

```
Hugo Static Site Generator v0.26 darwin/amd64 BuildDate: 2017-08-07T00:09:45-07:00
```

I'll see if I can figure out a good way to parse that, though.

Reply:
`| cut -f' ' -c5`

Reply:
or `| awk '{print $5}'`